src/Entity/Incentive.php

```diff
@@ -0,0 +1,365 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\IncentiveRepository;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass=IncentiveRepository::class)
+ */
+class Incentive
+{
+    public static $incentiveTypeChoices = [
+        'Cash' => 'cash',
+        'Gift Card' => 'gift_card',
+        'Voucher' => 'voucher',
+        'Promotional Item' => 'promotional',
+        'Other' => 'other'
+    ];
+
+    public static $incentiveOccurrenceChoices = [
+        'One-time Incentive' => 'one_time',
+        'Redraw' => 'redraw',
+        'Other' => 'other'
+    ];
+
+    public static $incentiveAmountChoices = [
+        '$25.00' => '25',
+        '$15.00' => '15',
+        'Other' => 'other'
+    ];
+
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=50)
+     */
+    private $participantId;
+
+    /**
+     * @ORM\OneToOne(targetEntity="App\Entity\User")
+     */
+    private $user;
+
+    /**
+     * @ORM\Column(type="string", length=50)
+     */
+    private $site;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $incentiveDateGiven;
+
+    /**
+     * @ORM\Column(type="string", length=50)
+     */
+    private $incentiveType;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $otherIncentiveType;
+
+    /**
+     * @ORM\Column(type="string", length=50)
+     */
+    private $incentiveOccurrence;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $otherIncentiveOccurrence;
+
+    /**
+     * @ORM\Column(type="string", length=50)
+     */
+    private $incentiveAmount;
+
+    /**
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    private $otherIncentiveAmount;
```

Comment:
I understand that this is following the other "other" patterns, but I'm wondering if this should be different. And maybe we talked about this a while ago but I didn't fully understand the situation here. It seems like the incentive amount should always be a number, and not only when "other" is selected. We can discuss more on Monday.